### ISSUE TITLE
VM Instances stuck

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp.dev#5346]
Noticed slow TTFB for amp.dev + a few failed healthchecks. This was caused by instance timeouts across all zones. The problematic instance template was: it-amp-dev-985a1a4-1612981927655. I've reverted to it-amp-dev-0ed6101-1612950462329 which seemed to have fixed it.

//cc @matthiasrohmer

**Comment by patrickkettner** [comment]
@sebastianbenz what is the course of action on this issue?

**Comment by sebastianbenz** [comment]
Discovered more details in the logs:

```
IOError: [Errno 28] No space left on device
at copyfile (/usr/lib64/python2.7/shutil.py:98)
at copy (/usr/lib64/python2.7/shutil.py:133)
at SetConfiguredUsers (/usr/lib64/python2.7/site-packages/google_compute_engine/accounts/accounts_utils.py:363)
at HandleAccounts (/usr/lib64/python2.7/site-packages/google_compute_engine/accounts/accounts_daemon.py:285)
at WatchMetadata (/usr/lib64/python2.7/site-packages/google_compute_engine/metadata_watcher.py:200)
```

**Comment by matthiasrohmer** [comment]
🤔  this is strange. Do our build artifacts already come close to the VMs disk space of 20GB by now?

There is no change I can recall that would write files to the disk at runtime. Can anyone else?

**Comment by sebastianbenz** [comment]
AMP Optimizer caches runtime data and validation rules on disk, but that
shouldn't be much.

>

**Comment by sebastianbenz** [comment]
I think I've just encountered the same problem when deploying `gcr.io/amp-dev-230314/amp-dev:56b92a0-1614805311229`:

```
Default
2021-03-03 22:20:23.598 UTC+1
"2021/03/03 21:20:23 Saving welcome script to profile.d "
Default
2021-03-03 22:20:23.598 UTC+1
"2021/03/03 21:20:23 Error: Failed to start container: Error: No such image: gcr.io/amp-dev-230314/amp-dev:56b92a0-1614805311229 "
Default
2021-03-03 22:20:23.594 UTC+1
"2021/03/03 21:20:23 Found 0 volume mounts in container it-amp-dev-56b92a0-1614805311229 declaration. "
Default
2021-03-03 22:20:23.594 UTC+1
"2021/03/03 21:20:23 No containers created by previous runs of Konlet found. "
```

https://console.cloud.google.com/logs/query;query=%2528resource.type%3D%22gce_instance%22%20AND%20resource.labels.instance_id%3D%22125525640808120902%22%2529%20OR%20%2528resource.type%3D%22global%22%20AND%20jsonPayload.instance.id%3D%22125525640808120902%22%2529?project=amp-dev-230314

**Comment by sebastianbenz** [comment]
This was caused by the disk images being too small. Fixed by #5475
